ext/stackprof/stackprof.c

```diff
@@ -617,7 +638,7 @@ stackprof_record_gc_samples()
         _stackprof.lines_buffer[1] = 0;
         _stackprof.unrecorded_gc_marking_samples--;
 
-        stackprof_record_sample_for_stack(2, timestamp_delta);
+        stackprof_record_sample_for_stack(2, start_timestamp, timestamp_delta);
```

Comment:
We end up using the same timestamp for each GC sample, which might look weird; maybe we should be spreading them out artificially the same way we do for time deltas. But that wasn't needed for my use case and this is simple.